### DISCUSSION TITLE
Some issues regarding the validation and comparison of results between coarse-grained molecular dynamics simulations using the Martini 3 force field and all-atom molecular dynamics simulations.

### DISCUSSION THREAD

**Discussion by zhangtrzhangtr** [ricalessandri/Martini3-small-molecules#50] [Q&A]
I am working on a system involving the evaporation of solvents from organic polymers. I have drawn on the evaporation method described in the paper you previously wrote, 'Bulk Heterojunction Morphologies with Atomistic Resolution from Coarse-Grain Solvent Evaporation Simulations'. In validating the molecular model designed with the Martini 3 force field, I used a method that involved placing a certain number of polymer chains, such as 50, in a chlorobenzene solvent, followed by 80 cycles of evaporation dynamics to produce a crystalline film of the polymer. The polymer chains were randomly divided into two groups, with 25 chains in each. I selected identical segments of atoms from each group to perform radial distribution functions to observe the stacking and aggregation of each polymer chain during the crystallization process. Using the same evaporation method and simulation time, I also performed radial distribution functions in all-atom molecular dynamics and noticed some differences in their aggregation and stacking. The coarse-grained model seemed more prone to organized crystalline stacking than the all-atom model used for parameterization. Is this a problem with the force field itself, my method of comparison, or perhaps an issue with how the models were constructed?  Thank you for your guidance."

**Comment by ricalessandri** [comment]
@zhangtrzhangtr interesting study, and very interesting topic! However, this is a rather complex problem. It is challenging to compare all-atom (AA) and CG propensity to crystallize. One needs to be careful with the timescales. The AA model's dynamics is much slower, and crystallization is an intrinsically slow process in itself, adding slowness to slowness. Since you use an evaporation protocol, and in particular you mention that you use the same *simulation time* for the AA and the CG models, the crystallization during the evaporation will just be much slower for the AA model, and the system will likely get kinetically trapped much earlier on than the CG model. Which seems to be what you are now observing. 

How much faster is the dynamics of the CG model w.r.t. to the AA model? This is not a trivial question to answer. As a number degrees of freedom are lost when coarse-graining, the resulting free energy landscape at the CG level gets smoothened out and dynamics gets faster. The coarser the model, the faster the dynamics. To go back to the crystallization propensity, what would be a far comparison is only if you reach equilibrium with both the CG model and the AA model and then compare these. Or somehow find a way to renormalize the dynamics of the model for a fair comparison. Once you are in that position, you could assess whether there's any problem/difference with the force field(s) (parametrization). I hope this helps and feel free to follow up.
